olive/passes/olive_pass.py

```diff
@@ -214,6 +211,25 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
         """
         raise NotImplementedError()
 
+    @classmethod
+    def data_configs(cls) -> Dict[str, PassDataConfigParam]:
```

Comment:
could we consider using _default_config if user would like explicitly specify the data_config?

Reply:
the _default_config seems a little bit duplicate with _data_config method?

Reply:
Adding to default_config is possible too but it makes it hard to get the names of data_config parameters (since we need accelerator config to use default_config) 
And it doesn’t have the option to whether to auto insert from input model or not.